### PR TITLE
BETA: Register xploite3r.is-a.dev

### PR DESCRIPTION
**Pull request by xploite3r**
Added `xploite3r.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/xploite3r.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "xploite3r",
+        "email": "xploite3r@gmail.com"
+    },
+    "record": {
+        "A": ["185.199.108.153"]
```
Comment (Stef-00012) [inline domains/xploite3r.json:7]: that ip seems to be for github pages, it requries 4 IPs
- 185.199.108.153
- 185.199.109.153
- 185.199.110.153
- 185.199.111.153

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/xploite3r.json (hunk 1 above)
Comment (xploite3r) [reply domains/xploite3r.json:7]: how do i fix this?

File: domains/unblokz.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "xploite3r",
+        "email": "xploite3r@gmail.com"
+    },
+    "record": {
+        "URL": "sites.google.com/view/unblokz"
```
Comment (Stef-00012) [inline domains/unblokz.json:7]: URL records must have the protocol (`https://`)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/xploite3r.json (hunk 1 above)
Comment (Stef-00012) [reply domains/xploite3r.json:7]: > how do i fix this?

just add the missing 3 IPs

Comment (xploite3r) [reply domains/xploite3r.json:7]: Can I add them through the pannel?

File: domains/sitestools.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "xploite3r",
+        "email": "xploite3r@gmail.com"
+    },
+    "record": {
+        "CNAME": "ns1.infinityfree.com ns2.infinityfree.com"
```
Comment (Stef-00012) [inline domains/sitestools.json:7]: You can't use a nameserver (NS) as CNAME

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**